src/index.html

```diff
@@ -4,9 +4,388 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="menu">
```

Comment:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/212480937-0ff1b580-2e58-4c3b-8fe9-3544c2a825c2.png">

```suggestion
      </a>
      
      <nav class="menu">
```